parlai/agents/hugging_face/dict.py

```diff
@@ -16,11 +16,11 @@
 
 
 try:
-    from transformers import GPT2Tokenizer, T5TokenizerFast
+    from transformers import GPT2Tokenizer, T5TokenizerFast, LlamaTokenizer
```

Comment:
Is there a `LlamaTokenizerFast` too?

Reply:
you got me LOL. I marked it as todo. I'll check if that work too